rfcs/0001-authorisation/README.md

```diff
@@ -0,0 +1,166 @@
+# RFC-0001 Memorandum on Flux Authorisation
```

Comment:
Can you please use "Authorization" to match [Kubernetes docs](https://kubernetes.io/docs/reference/access-authn-authz/authorization/), I know it's not proper UK english...